pkg/extensions/enterprise_imports.go

```diff
@@ -53,5 +53,6 @@ import (
 	_ "github.com/grafana/e2e"
 	_ "github.com/grafana/gofpdf"
 	_ "github.com/grafana/gomemcache/memcache"
+	_ "github.com/grafana/grafana/apps/alerting/alertenrichment/pkg/apis/alertenrichment/v1beta1"
```

Comment:
Is this relevant to the PR?

Reply:
No, but somehow this was removed and needed to be re-added. I would guess it's an artifact of some bad merge and might be taken care of when we re-update and squash

Reply:
This is actually in the [latest main](https://github.com/grafana/grafana/blob/ebd1f63aab70387c1bf0bd9cfe4be9149abad015/pkg/extensions/enterprise_imports.go#L56). 🤔  But yes, recently someone removed it in an unrelated PR by accident, and I had to add it again.